tests/test_client.py

```diff
@@ -174,6 +174,25 @@ def test_step005_apl_query(self):
 
         self.assertEqual(len(qr.matches), len(self.events))
 
+    def test_step005_apl_query_messages(self):
+        """Test an APL query with messages"""
+        startTime = datetime.utcnow() - timedelta(minutes=2)
+        endTime = datetime.utcnow()
+
+        apl = "['%s'] | where true" % self.dataset_name
+        opts = AplOptions(
+            start_time=startTime,
+            end_time=endTime,
+            format=AplResultFormat.Legacy,
+        )
+        qr = self.client.query(apl, opts)
+        # "where clause always evaluates to TRUE, which will include all data"
+        self.assertEqual(len(qr.status.messages), 1)
+        self.assertEqual(
+            qr.status.messages[0].msg,
+            "line: 1, col: 24: where clause always evaluates to TRUE, which will include all data",
+        )
```

Comment:
I'm unfamiliar with your test set up / fixtures.
Does this query a "real" Axiom server?

Are you seeing the `apl_whereclausealwaysevaluatestoTRUEwhichwillincludealldata_5` string in the response? If not, maybe something changed in the backend/server?